src/components-examples/cdk-experimental/listbox/cdk-listbox/cdk-listbox-example.css

```diff
@@ -0,0 +1,60 @@
+.example-listbox-controls {
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  padding-bottom: 16px;
+}
+
+ul {
```

Comment:
nit: I generally push for all example styles to be defined as CSS classes that start with `.example-` so that it's clear to the developer what's an example style vs. a system or component style